values.yaml

```diff
@@ -203,7 +203,7 @@ doc_pyincore_viz:
   replicas: 1
   image:
     repository: doc/pyincore-viz
-    tag: 1.11.0
+    tag: v1.11.0
```

Comment:
v needed?

Reply:
yes, unfortunately. There is an error in github action so all of them (pyincore, viz, data) can't produce the tag without v. But v tagged one is being produced when the release gets created

Reply:
I made PR for fix this one for pyincore-viz. If it get fixed, will do the same for pyincore, and data